assets/component-slider.css

```diff
@@ -359,6 +363,43 @@ slider-component.slider-component-full-width {
   justify-content: center;
 }
 
+.announcement-bar .slider-button--next {
+  margin-right: -1.6rem;
+  height: 100%;
+}
+
+.announcement-bar .slider-button--prev {
+  margin-left: -1.6rem;
+  height: 100%;
+}
+
+.announcement-bar .slider-button--next:focus-visible,
+.announcement-bar .slider-button--prev:focus-visible {
+  outline-offset: -0.3rem;
+  box-shadow: 0 0 0 -0.2rem rgb(var(--color-foreground));
+}
+
+@media screen and (min-width: 750px) {
+  .announcement-bar .slider-button--next {
+    margin-right: -3.8rem;
+  }
+
+  .announcement-bar .slider-button--prev {
```

Comment:
Not something we need to address here or at all, just thinking aloud in case it appealed to anyone. With the arrows kind of serving as a frame, it might be a nice little improvement if the boundaries of the slider area had a little fade out/gradient overlay so the scrolling content doesn't just disappear at a harsh line. Something similar may have come up before though (maybe during slideshow development) and I know we didn't roll with that.
<img src="https://screenshot.click/30-47-fhmyg-xyzsf.png" width="300" />